js/package.json

```diff
@@ -7,6 +7,7 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest --silent",
+    "teste-snapshot-update": "yarn test -u",
```

Comment:
I think you can add a rule in the Makefile, instead of a script in the `package.json`